### DISCUSSION TITLE
Could `redirects` work with Prerender?

### DISCUSSION THREAD

**Discussion by EralChen** [vikejs/vike#2000] [Discussion]
![image](https://github.com/user-attachments/assets/cd3ce2f2-5cd3-4cfc-9978-b8bdaf5100f0)

I set this in my `vite.config.ts` , it works on dev(vite).

but `dist/client` on github-pages,

![image](https://github.com/user-attachments/assets/bdd91d84-b0f9-43c6-b473-c6d2de6ccc1e)

it  seems `redirects` does't work

**Comment by brillout** [comment]
I believe you should be getting a warning telling you that it isn't supported. It's on the radar: https://github.com/vikejs/vike/issues/2001.
